client/src/pages/Profile/useStyles.ts

```diff
@@ -0,0 +1,66 @@
+import { makeStyles } from '@material-ui/core/styles';
+const useStyles = makeStyles((theme) => ({
+  profileContainer: {
+    margin: theme.spacing(4),
+    height: '900px',
+    maxWidth: '760px',
+  },
+  coverImage: {
+    maxHeight: '260px',
+    width: '760px',
```

Comment:
I was wondering if this works well on smaller screens where the container might be smaller than `760px`.

Reply:
I have changed it to 100% so it changes according to its container. Thanks for that :)